pkgs/os-specific/linux/projecteur/default.nix

```diff
@@ -10,27 +17,38 @@ mkDerivation rec {
     repo = "Projecteur";
     rev = "v${version}";
     fetchSubmodules = false;
-    sha256 = "sha256-kg6oYtJ4H5A6RNATBg+XvMfCb9FlhEBFjfxamGosMQg=";
+    hash = "sha256-kg6oYtJ4H5A6RNATBg+XvMfCb9FlhEBFjfxamGosMQg=";
   };
 
   postPatch = ''
     sed '1i#include <array>' -i src/device.h # gcc12
+    substituteInPlace 55-projecteur.rules.in \
+      --replace 'KERNEL=="uinput", SUBSYSTEM=="misc", TAG+="uaccess", OPTIONS+="static_node=uinput' 'KERNEL=="uinput", SUBSYSTEM=="misc", TAG+="uaccess", OPTIONS+="static_node=uinput, GROUP="input", MODE="0660"'
```

Comment:
I think it might be better to use a patch instead of substitude inplace (of the linked PR) or wait for upstream to include the patch.

Reply:
Done :)